Assets/Javascript/storage.js

```diff
@@ -0,0 +1,44 @@
+const valuesArray = [];
+valuesArray.push([document.getElementById('name_input'), 'nameInputValue']);
+valuesArray.push([document.getElementById('email_input'), 'emailInputValue']);
+valuesArray.push([document.getElementById('contact_text_area'), 'contactTextAreaValue']);
+
+function loadValues() {
+  for (let i = 0; i < valuesArray.length; i += 1) {
+    if (localStorage.getItem(valuesArray[i][1])) {
+      valuesArray[i][0].value = localStorage.getItem(valuesArray[i][1]);
+    }
+  }
+}
```

Comment:
- [ ] Please, The form data for all inputs should be stored in a single object, saved to a single local storage registry. 🐱 

```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```